internal/templateUtils/template_test.go

```diff
@@ -0,0 +1,8 @@
+package templateUtils
+
+import "testing"
+
+func TestGetCIDR(t *testing.T) {
+	cidr := GetCIDR("10.0.0.0/16", 2, 1)
+	t.Log(cidr)
+}
```

Comment:
Do we even need a test that will never fail?

Reply:
This test was used to check if CIDR changes without running it in the template. We can remove it at this point, if you wish.

Reply:
actually lets keep it, I don't mind maybe it might catch some errors in future